OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -260,9 +260,9 @@ public void UpdateMouseover()
 				}
 			}
 
-			if (resourceLayer != null)
+			if (resourceRenderer != null)
 			{
-				var resource = resourceLayer.GetRenderedResource(cell);
+				var resource = resourceRenderer.GetRenderedResourceType(cell);
```

Comment:
And then this remains as it was.

> 1

Reply:
`resourceLayer.GetRenderedResource` is gone. That's kind of the point of this change.